src/app/sportovni-akce/[category]/[id]/EventDetail/TableRow.tsx

```diff
@@ -1,9 +1,11 @@
+import { FC, ReactNode } from 'react';
+
 type TableRowProps = {
   label: string;
-  value: React.ReactNode;
+  value: ReactNode;
 };
 
-export const TableRow: React.FC<TableRowProps> = ({ label, value }) => (
+export const TableRow: FC<TableRowProps> = ({ label, value }) => (
```

Comment:
I am sorry I said it wrong. React.memo and React.FC is the only prefixed one (I like). I am fixing it myself! Sorry for that.